packages/eslint-plugin-pf-codemods/src/rules/v6/expandableSectionRemoveIsActiveProp/expandableSection-remove-isActive-prop.test.ts

```diff
@@ -0,0 +1,25 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./expandableSection-remove-isActive-prop";
+
+ruleTester.run("expandableSection-remove-isActive-prop", rule, {
+  valid: [
+    {
+      code: `<ExpandableSection isActive />`,
+    },
+    {
+      code: `import { ExpandableSection } from '@patternfly/react-core'; <ExpandableSection someOtherProp />`,
+    },
+  ],
+  invalid: [
```

Comment:
One thing we typically try to do is include tests where the import path is `@patternfly-react/dist/[esm|js|dynamic]/...` (the dualListSelectorNextPromoted rule has such tests). Would you mind just adding those in here as well? Everything else in the test can remain the same as this one, just the import path getting updated.

Reply:
added, when in development i didn't see many codemods that do that, at least not consistently, but now that i revisit i do see a few. easy enough to do, thanks.

Reply:
Yeah to be fair it's something we intend to go back to to ensure they all do that, so it's more a "if we remember to we include them" rule 😆